PythonScripts/UniFolderCreator/UniFolderCreator.py

```diff
@@ -1,10 +1,15 @@
 import os
+import sys
 from sys import exit
 from pathlib import Path
 
 # Directory path to wherever you have your folder for the semester.
 # Eg 'C:\Users\Tim\Google Drive\2023 Sem 2'
-FOLDER_DIR: Path = Path.cwd()
+try:
+    FOLDER_DIR: Path = Path(sys.argv[1])
```

Comment:
I think it makes more sense to put this in the `main` function so that people could still call this as a module from another python file, but it's not critical for this script.

Reply:
I agree, just put it in main